common-lib/src/main/scala/com/gu/mediaservice/model/MimeType.scala

```diff
@@ -0,0 +1,41 @@
+package com.gu.mediaservice.model
+
+import play.api.Logger
+import play.api.libs.json._
+
+class UnsupportedMimeTypeException extends Exception
+
+sealed trait MimeType {
+  def name: String = this match {
+    case Jpeg => "image/jpeg"
+    case Png => "image/png"
+    case Tiff => "image/tiff"
+  }
+
+  def fileExtension: String = name.split('/').reverse.head
```

Comment:
Not that it matters much on output, but we currently produce eg.  a `jpg` extension. If that’s ever (to be) used on input, it would better take into account different spellings, though.

Reply:
Sorry, ignore, saw [line 37](https://github.com/guardian/grid/pull/2742/files/2fbd965cc640d9965dffb1471bebc6c3e9a000eb#diff-214fa5d666a63a722162df9e56a4968dR37).

Reply:
So this would only recognise `.jpg` as a JPEG then?

Reply:
1. I hope we don’t do recognition by extension (we have Tika and metadata-extractor has some shiffing too), but wanted to double-check.
2. I thought this is about what extension we add when producing JPEGs.

Reply:
Correct, `fileExtension` is only used for the output of a crop in S3, not for representing mimetype.